42196 Add text about Umbraco.Cloud.Cms package (#6368)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/availability-performance.md

For cloud projects with a dedicated option (or shared plans with another granularity than 5 minutes), users will see the average assigned private bytes in bytes. Here the statistics panel will display the maximum, average, and minimum allocation of private bytes based on selected granularity.

#### Platform and CMS events

> The charts are enhanced with platform event like restarts, automatic and manual upgrades, environment-to-environment deployments and plan changes.

Reviewer: ```suggestion
The charts are enhanced with platform events like restarts, automatic and manual upgrades, environment-to-environment deployments, and plan changes.
```

> This information will assist you in potential troubleshooting, make informed decisions, and ensure smooth project management.

Reviewer: ```suggestion
This information will help you in potential troubleshooting, make informed decisions, and ensure smooth project management.
```

By utilizing the `Umbraco.Cloud.Cms` package we are tracking the hot and colds boots of your Umbraco environment in cloud.

> The package is installed on v10, v13 and v14 environments in Umbraco Cloud.

Reviewer 2: Can we future proof this line? make itsomething like on v10, v13, v14 and all new and upcoming versions

Author: I've changed it a bit + and moved one bit around - u like?

Reviewer 3: How about:
"The package is installed on all new and upcoming versions of Umbraco, incl. versions 10, 13, and 14."

> You can read more about [Hot vs. Cold restarts here](../../../14/umbraco-cms/reference/notifications/hot-vs-cold-restarts.md).

Reviewer: ```suggestion
Read about [Hot vs. Cold restarts](https://docs.umbraco.com/umbraco-cms/reference/notifications/hot-vs-cold-restarts).
```

> The package is installed on v10, v13 and v14 environments in Umbraco Cloud. The package will be part of all new Cloud projects for all upcoming versions of the CMS.

Reviewer: ```suggestion
The package is installed on all environments running Umbraco 10, 13, and 14 on Umbraco Cloud. The package will be part of new Cloud projects on upcoming versions of Umbraco CMS.
```
